coops-ui/tests/unit/TodoList.spec.ts

```diff
@@ -51,6 +58,9 @@ describe('TodoList - UI', () => {
               'name': 'annualReport',
               'ARFilingYear': 2017,
               'status': 'NEW'
```

Comment:
Could status be initialized from enum?

Reply:
absolutely, i must have missed one. Thanks!

Reply:
Ohh, these are the in spec file! I'm not sure whats the way to go here, in my opinion i don't mind using the string values at all in tests as they can also double as a validation of the enums/constants used in the code. What are your thoughts on that?

Reply:
Good point! I agree with you.